Redux patterns/redux+typescript.md

```diff
@@ -297,21 +297,21 @@ const ConnectedAddTodo = ({ dispatch }) => {
       type: "ADD_TODO", 
       text: input.current.value
     })
-  }}        
```

Comment:
this is just removing extra spaces. Not important I know but it's just a habit for me